src/main/java/org/jabref/logic/util/io/FileNameCleaner.java

```diff
@@ -18,18 +18,16 @@
             10, 11, 12, 13, 14, 15, 16, 17, 18, 19,
             20, 21, 22, 23, 24, 25, 26, 27, 28, 29,
             30, 31, 34,
-            42, 47,
```

Comment:
Why did you remove 47 here? Why is it no longer an illegal char? This needs a good justification or should be reverted.

Reply:
See comment in my e-mail below.